qek/data/graphs.py

```diff
@@ -197,9 +197,24 @@ def compute_register(self) -> pl.Register:
         assert pos is not None
         return pl.Register.from_coordinates(coords=pos)
 
-    def compute_sequence(self) -> pl.Sequence:
+    # Default values for the sequence.
+    #
+    # See the companion paper for an explanation.
+    SEQUENCE_DEFAULT_AMPLITUDE_RAD_PER_US = 1.0 * 2 * np.pi
+    SEQUENCE_DEFAULT_DURATION_NS = 660
+
+    def compute_sequence(
+        self, amplitude: float = SEQUENCE_DEFAULT_AMPLITUDE_RAD_PER_US, duration: int = 660
```

Comment:
No use for the `SEQUENCE_DEFAULT_DURATION_NS` here ?

Reply:
Oops!